src/Makefile.qt.include

```diff
@@ -234,6 +237,11 @@ RES_ICONS = \
   qt/res/icons/bitcoin.ico \
   qt/res/icons/bitcoin_testnet.ico \
   qt/res/icons/bitcoin.png \
+  qt/res/icons/button_off.png \
+  qt/res/icons/button_on_blue.png \
```

Comment:
@jonasschnelli What do you think about using `QCheckBox` with `setPallette()` as suggested by @luke-jr ?